.lintstagedrc

```diff
@@ -0,0 +1,6 @@
+{
+  "*.{ts,tsx}": ["prettier --write --config .prettierrc"],
+  "{*.json,.huskyrc,.prettierrc,.lintstagedrc,.eslintrc}": [
+    "prettier --write --parser json --config .prettierrc"
+  ]
+}
```

Comment:
this should not be in this pr
use https://github.com/dpolevodin/js_school_diary/pull/11

Reply:
If they fix it, I'll remove it.

Reply:
any way this should not be in other pr (like feat(lint))